<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoccerView</class>
 <widget class="QMainWindow" name="SoccerView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VSSReferee</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="FieldView" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>600</width>
      <height>420</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="scoreboard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>610</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Group Phase</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="leftTeamLogo">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>50</y>
       <width>96</width>
       <height>96</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:60px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/teams/default.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="rightTeamLogo">
     <property name="geometry">
      <rect>
       <x>509</x>
       <y>50</y>
       <width>96</width>
       <height>96</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:60px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/teams/default.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="leftTeamGoals">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeMono</family>
       <pointsize>58</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="rightTeamGoals">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeMono</family>
       <pointsize>58</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gameTime">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>70</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeMono</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red; 
border-radius:10px;
color:white;</string>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="halfTime">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>30</y>
       <width>398</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NO_HALF</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="statusColor">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>130</y>
       <width>398</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red; 
border-radius:10px;
color: white;</string>
     </property>
     <property name="text">
      <string>xxxxxxxxxxxxx</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="leftTeamName">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>27</y>
       <width>96</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Default Team</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="rightTeamName">
     <property name="geometry">
      <rect>
       <x>509</x>
       <y>27</y>
       <width>96</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Default Team</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="sendToDiscord">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>104</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Send to Discord</string>
     </property>
    </widget>
    <zorder>statusColor</zorder>
    <zorder>leftTeamLogo</zorder>
    <zorder>rightTeamLogo</zorder>
    <zorder>leftTeamGoals</zorder>
    <zorder>rightTeamGoals</zorder>
    <zorder>gameTime</zorder>
    <zorder>halfTime</zorder>
    <zorder>leftTeamName</zorder>
    <zorder>rightTeamName</zorder>
    <zorder>sendToDiscord</zorder>
   </widget>
   <widget class="QGroupBox" name="manualReferee">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>421</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Manual Referee</string>
    </property>
    <widget class="QPushButton" name="haltGame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>95</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>HALT</string>
     </property>
     <property name="text">
      <string>Halt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forYellow">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>140</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>YELLOW</string>
     </property>
     <property name="text">
      <string>Yellow</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">forButtons</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="startGame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>95</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>GAME_ON</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addGoal">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Goal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addMinute">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Min</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addSecond">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add sec</string>
     </property>
    </widget>
    <widget class="QPushButton" name="q1">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>230</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>QUADRANT_1</string>
     </property>
     <property name="text">
      <string>Q1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">quadrantButtons</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="forBlue">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>170</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>BLUE</string>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">forButtons</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="command">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>120</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Command</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="freeball">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>FREE_BALL</string>
     </property>
     <property name="text">
      <string>Free_Ball</string>
     </property>
    </widget>
    <widget class="QPushButton" name="q3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>260</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>QUADRANT_3</string>
     </property>
     <property name="text">
      <string>Q3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">quadrantButtons</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="penaltyKick">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>170</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>PENALTY_KICK</string>
     </property>
     <property name="text">
      <string>Penalty_Kick</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goalKick">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>GOAL_KICK</string>
     </property>
     <property name="text">
      <string>Goal_Kick</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kickoff">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>140</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>KICKOFF</string>
     </property>
     <property name="text">
      <string>Kickoff</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rmvGoal">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>200</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Rmv Goal</string>
     </property>
    </widget>
    <widget class="QLabel" name="atQuadrant">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>210</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Quadrant</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="q2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>230</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>QUADRANT_2</string>
     </property>
     <property name="text">
      <string>Q2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">quadrantButtons</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="q4">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>260</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>QUADRANT_4</string>
     </property>
     <property name="text">
      <string>Q4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">quadrantButtons</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="gControl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Game Control</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="forTeam">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>120</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>For team</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="stopGame">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>50</y>
       <width>95</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>STOP</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="penaltyShootout">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>226</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Penalty Shootout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rmvField">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>226</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>WO Kickoff</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="refereeSuggestions">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>5</y>
      <width>421</width>
      <height>135</height>
     </rect>
    </property>
    <property name="title">
     <string>Referee suggestions</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>15</y>
       <width>421</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="suggestionGrid"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="teleportBox">
      <property name="geometry">
            <rect>
                  <x>630</x>
                  <y>210</y>
                  <width>421</width>
                  <height>65</height>
            </rect>
      </property>
      <property name="title">
            <string>Teleport</string>
      </property>
      <widget class="QPushButton" name="enableTeleport">
            <property name="geometry">
            <rect>
                  <x>110</x>
                  <y>30</y>
                  <width>90</width>
                  <height>25</height>
            </rect>
            </property>
            <property name="whatsThis">
                  <string>ENABLE TELEPORT</string>
            </property>
            <property name="text">
                  <string>Enable</string>
            </property>
            <property name="checkable">
                  <bool>true</bool>
            </property>
            <property name="checked">
                  <bool>false</bool>
            </property>
      </widget>
      <widget class="QPushButton" name="disableTeleport">
            <property name="geometry">
            <rect>
                  <x>210</x>
                  <y>30</y>
                  <width>90</width>
                  <height>25</height>
            </rect>
            </property>
            <property name="whatsThis">
                  <string>DISABLE TELEPORT</string>
            </property>
            <property name="text">
                  <string>Disable</string>
            </property>
            <property name="checkable">
                  <bool>true</bool>
            </property>
            <property name="checked">
                  <bool>true</bool>
            </property>
      </widget>
   </widget>
   <widget class="QGroupBox" name="visionBox">
      <property name="geometry">
            <rect>
                  <x>630</x>
                  <y>145</y>
                  <width>421</width>
                  <height>65</height>
            </rect>
      </property>
      <property name="title">
            <string>Vision Packets</string>
      </property>
      <widget class="QPushButton" name="SSLVision">
            <property name="geometry">
            <rect>
                  <x>80</x>
                  <y>30</y>
                  <width>120</width>
                  <height>25</height>
            </rect>
            </property>
            <property name="whatsThis">
                  <string>SSL VISION</string>
            </property>
            <property name="text">
                  <string>SSLvision packet</string>
            </property>
            <property name="checkable">
                  <bool>true</bool>
            </property>
            <property name="checked">
                  <bool>false</bool>
            </property>
      </widget>
      <widget class="QPushButton" name="FIRAVision">
            <property name="geometry">
            <rect>
                  <x>230</x>
                  <y>30</y>
                  <width>120</width>
                  <height>25</height>
            </rect>
            </property>
            <property name="whatsThis">
                  <string>FIRA VISION</string>
            </property>
            <property name="text">
                  <string>FIRA packet</string>
            </property>
            <property name="checkable">
                  <bool>true</bool>
            </property>
            <property name="checked">
                  <bool>true</bool>
            </property>
      </widget>
   </widget>
   <widget class="QLabel" name="flag">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FieldView</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">src/soccerview/fieldview/fieldview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="forButtons"/>
  <buttongroup name="quadrantButtons"/>
 </buttongroups>
</ui>
